packages/grafana-ui/src/components/Icon/iconBundle.ts

```diff
@@ -1,7 +1,13 @@
 /* eslint-disable import/order */
 // DO NOT EDIT THIS FILE
 // This file is automatically generated  (do not edit it here)
-// see @grafana/ui/scripts/generate-icon-bundle.js
+//
+// After adding icon to correct directory and to
+// `packages/grafana-ui/src/components/Icon/cached.json`,
+// see @grafana/ui/scripts/generate-icon-bundle.js.
+// To run the generateIconBundle function, change directories in terminal
+// to grafana-ui, run `yarn generate-icon-bundle`, then in code editor add
+// icon key to `availableIconsIndex` const in `packages/grafana-data/src/types/icon.ts`
```

Comment:
This is a great addition but I think you need to add it to the template. Otherwise this will be overwritten next time someone regenerate this file.

The template exists in: `packages/grafana-ui/src/components/Icon/iconBundle.ts.template`

Reply:
+1 with marcus. This is an automatically generated file (as indicated in the comment before this one).

Reply:
Thanks I wasnt sure if that applied to the comments as well. Updated!

Reply:
Awesome, now when you have that in place. Please re-run the icon generation and the `iconBundle.ts` should be updated with the new comment.